packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -795,47 +766,108 @@ export class SDKClient {
    * @param callerName
    * @param interactingEntity
    */
-  public deleteFile = async (fileId: FileId, requestId?: string, callerName?: string, interactingEntity?: string) => {
+  public deleteFile = async (fileId: FileId, requestId: string, callerName: string, interactingEntity: string) => {
     // format request ID msg
     const currentDateNow = Date.now();
     const requestIdPrefix = formatRequestIdMessage(requestId);
+    let fileDeleteTx: FileDeleteTransaction | null = null;
 
     try {
       // Create fileDeleteTx
-      const fileDeleteTx = new FileDeleteTransaction()
+      fileDeleteTx = new FileDeleteTransaction()
         .setFileId(fileId)
         .setMaxTransactionFee(new Hbar(2))
         .freezeWith(this.clientMain);
 
       // execute fileDeleteTx
-      const fileDeleteTxResponse = await fileDeleteTx.execute(this.clientMain);
+      await fileDeleteTx.execute(this.clientMain);
```

Comment:
Same here - don't we want to use `executeTransaction` here and also for all other transactions executed inside `SdkClient`?

Reply:
We can also add a method `executeAllTransactions` and reuse it for cases where we want to call `executeAll`. 

In this way we can only define this `try`/`catch`/`finally` logic only in one place and we don't have to duplicate it everywhere, just have to make sure all transactions and queries are using `executeTransaction`/`executeAllTransactions` and `executeQuery`.